src/libstd/net/addr.rs

```diff
@@ -142,8 +201,14 @@ impl SocketAddr {
         }
     }
 
-    /// Returns true if the IP in this `SocketAddr` is a valid IPv4 address,
-    /// false if it's a valid IPv6 address.
+    /// Returns [`true`] if the [IP address] in this `SocketAddr` is an
+    /// [IPv4 address] and [`false`] if it's an [IPv6 address].
```

Comment:
One formulation I thought about was ``, `false` otherwise``, maybe that's better?